src/test/java/io/stargate/sgv2/jsonapi/api/v1/FindCollectionsIntegrationTest.java

```diff
@@ -68,11 +74,88 @@ public void happyPath() {
           .post(NamespaceResource.BASE_PATH, namespaceName)
           .then()
           .statusCode(200)
-          .body("status.collections", hasSize(1))
-          .body("status.collections", hasItem(collection));
+          .body("status.collections", hasSize(greaterThanOrEqualTo(1)))
+          .body("status.collections", hasItem("collection1"));
     }
 
     @Test
+    @Order(1)
+    public void happyPathWithExplain() {
+      String json =
+          """
+              {
+                "createCollection": {
+                  "name": "%s",
+                  "options": {
+                    "vector": {
+                      "size": 5,
+                      "function": "cosine"
+                    }
+                  }
+                }
+              }
+              """
+              .formatted("collection2");
+
+      given()
+          .header(HttpConstants.AUTHENTICATION_TOKEN_HEADER_NAME, getAuthToken())
+          .contentType(ContentType.JSON)
+          .body(json)
+          .when()
+          .post(NamespaceResource.BASE_PATH, namespaceName)
+          .then()
+          .statusCode(200)
+          .body("status.ok", is(1));
+
+      String expected1 =
+          """
+                  {
+                    "name": "%s"
+                    }
+                  }
+                    """
+              .formatted("collection1");
+      String expected2 =
+          """
+              {
+                  "name": "%s",
+                  "options": {
+                    "vector": {
+                      "size": 5,
+                      "function": "cosine"
+                    }
+                  }
+                }
+                """
+              .formatted("collection2");
+
+      json =
+          """
+              {
+                "findCollections": {
+                  "options": {
+                    "explain" : true
```

Comment:
What happens if you don't pass `explain` here? Do you not get the `options` back?

Reply:
Yes., it will only return the collection names.